apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -576,6 +577,34 @@ data(_) ->
     emqx_ctl:usage([{"data import <File>",   "Import data from the specified file"},
                     {"data export",          "Export data"}]).
 
+%%--------------------------------------------------------------------
+%% @doc acl Command
+
+acl(["cache-clean", "node", Node]) ->
+    case emqx_mgmt:clean_acl_cache_all(erlang:list_to_existing_atom(Node)) of
+        ok ->
+            emqx_ctl:print("ACL cache drain started on node ~s.~n", [Node]);
+        {error, Reason} ->
+            emqx_ctl:print("ACL drain failed on node ~s: ~0p.~n", [Node, Reason])
+    end;
+
+acl(["cache-clean", "all"]) ->
+    case emqx_mgmt:clean_acl_cache_all() of
+        ok ->
+            emqx_ctl:print("Started ACL cache drain in all nodes~n");
+        {error, Reason} ->
+            emqx_ctl:print("ACL cache-clean failed: ~p.~n", [Reason])
+    end;
```

Comment:
Then we should remove this clause. A CLI should only affect the current node.

Reply:
how about rest requests? if we limit only to "current" node it could lead to a situation, where acl cache is inconsistent throughout the cluster. as a devops (which im not) id like to avoid such situation as much as i can, as it could lead to different behavior for user depending on node one's connected to (ex. when in round robin balancing mode). 
tl;dr clearing only one node in cluster may be confusing.

Reply:
We have all our CLI interfaces operates only on local node.
Many REST APIs make RPC calls to nodes in the cluster.

I am not sure if there is a good reason why CLI should be local only.
If not, I'm for breaking the rule, if not now, we'll never have the chance until we deprecate the current CLI.
--- for the context, our long term plan is to wrap CLI around REST API

Reply:
I agree with @zmstone, so I have no any more comments on this PR.